File: Java/src/com/hackerrank/daysofcode/Operators.java
package com.hackerrank.daysofcode;

/**
 * Created by vmurakami on 18/07/17.
 */

import java.util.*;

    public class Operators {

        public static void main(String[] args) {
            Scanner scan = new Scanner(System.in);
            double mealCost = scan.nextDouble(); // original meal price
            int tipPercent = scan.nextInt(); // tip percentage
            int taxPercent = scan.nextInt(); // tax percentage
            scan.close();

            // Write your calculation code here.
            double numberToRoundHere = mealCost + (mealCost * tipPercent / 100) + (mealCost * taxPercent / 100);

            // cast the result of the rounding operation to an int and save it as totalCost
            int totalCost = (int) Math.round(numberToRoundHere);

            // Print your result
            System.out.println("The total meal cost is " + totalCost + " dollars.");
        }
    }

File: Scala/src/main/scala/Operators.scala
package main.scala

/**
  * Created by vmurakami on 19/07/17.
  */
object Operators {
  def main(args: Array[String]) {
    val mealCost = scala.io.StdIn.readDouble()
    val tipPercent = scala.io.StdIn.readInt()
    val taxPercent = scala.io.StdIn.readInt()
    val tip = mealCost * tipPercent / 100
    val tax = mealCost * taxPercent / 100
    val calculation = mealCost + tax + tip
    val totalCost = math.round(calculation)

    println(s"The total meal cost is $totalCost dollars.")
  }
}
